src/Compilers/CSharp/Portable/Symbols/Source/ExplicitInterfaceHelpers.cs

```diff
@@ -17,21 +17,74 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
+#nullable enable
+    internal class ExplicitInterfaceMemberInfo(ExplicitInterfaceSpecifierSyntax explicitInterfaceSpecifier, string name)
+    {
+        private TypeSymbol? _lazyExplicitInterfaceType;
+        private string? _lazyMemberMetadataName;
+
+        public ExplicitInterfaceSpecifierSyntax ExplicitInterfaceSpecifier => explicitInterfaceSpecifier;
```

Comment:
>explicitInterfaceSpecifier [](http://example.com/codeflow?start=78&length=26)

It looks like we are still holding onto syntax. Symbols are not supposed to hold onto syntax nodes.